docs/theme/main.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block announce %}
+  <a href="https://discordapp.com/channels/280102180189634562/645268178397560865">
+    Join us on
+    <span class="twemoji discord">
+      {% include ".icons/fontawesome/brands/discord.svg" %}
+    </span>
+    <strong>Discord</strong>
+  </a>
```

Comment:
I think one has to use the invite link to join the server as explained in:

https://github.com/sublimelsp/LSP/blob/9d83fee1020cd70ffc46c6f5b4b16e78500fe763/README.md?plain=1#L50

so it's probably worth including the invitation link here also.